packages/react-combobox/src/components/Combobox/renderCombobox.tsx

```diff
@@ -1,13 +1,24 @@
 import * as React from 'react';
+import { Portal } from '@fluentui/react-portal';
 import { getSlots } from '@fluentui/react-utilities';
 import type { ComboboxState, ComboboxSlots } from './Combobox.types';
+import { ComboboxContext } from '../../contexts/ComboboxContext';
+import { ComboboxContextValues } from '../../contexts/ComboboxContext';
 
 /**
  * Render the final JSX of Combobox
  */
-export const renderCombobox_unstable = (state: ComboboxState) => {
+export const renderCombobox_unstable = (state: ComboboxState, contextValues: ComboboxContextValues) => {
   const { slots, slotProps } = getSlots<ComboboxSlots>(state);
 
-  // TODO Add additional slots in the appropriate place
-  return <slots.root {...slotProps.root} />;
+  const dropdown = <slots.listbox {...slotProps.listbox}>{slotProps.root.children}</slots.listbox>;
+
+  return (
+    <slots.root {...slotProps.root}>
+      <ComboboxContext.Provider value={contextValues.combobox}>
+        <slots.trigger {...slotProps.trigger} />
+        {state.open ? state.inline ? dropdown : <Portal>{dropdown}</Portal> : null}
```

Comment:
_nit:_ Can we refactor this somehow? From my POV this nested ternary is not readable...